src/explore-education-statistics-common/src/components/util/__tests__/generateContentList.test.tsx

```diff
@@ -0,0 +1,56 @@
+import generateContentList from '@common/components/util/generateContentList';
+
+describe('Generate content list', () => {
+  test('Outputs a nested list', () => {
+    const fakeElement = document.createElement('div');
```

Comment:
We might possibly be able to do this with the testing-library `render()` function next time e.g.

```
render(
    <div>
       <h3 id="1">Heading 1</h3>
       <h3 ........>
       .......
    </div>
);
```

Reply:
Cool, I'll add that to my list of things to learn about :)